src/ant/doc/README.md

```diff
@@ -0,0 +1,39 @@
+# Antenna Rule Checker
+
+This tool checks antenna violations based on an input LEF and DEF file, a report
+will be generated to indicate violated nets. 4 APIs are provided as an interface with FastRoute to preemptively fix antenna violation as well as used for the diode insertion flow:
+
+## Antenna check Commands
+
+---
+
+ - `load_antenna_rules`: import antenna rules to ARC, must be called before other commands
+ - `check_antennas`: check antenna violations on all nets and generate a report
+   - -path: specify the path to save the report
+   - -simple_report: provides a summary of the violated nets 
+## Antenna fix Commands
+
+---
+
+### Tcl commands
+
+ - `check_net_violation`: check if an input net is violated, return 1 if the net is violated
+   - -net_name: set the net name for checking
+ - `find_max_wire_length`: print the longest wire in the design
+
+### C++ commands
```

Comment:
what is the point of documenting c++ "commands"? they are not commands in any sense

Reply:
Removed